samples/word/basics/get-built-in-properties.yaml

```diff
@@ -0,0 +1,95 @@
+﻿id:''
+gist: ''
+source: WORD
+author: Microsoft
+name: Get built-in document properties.
+description: 'Shows how to get built-in document properties.'
+script:
+  content: |+
+    $("#run").click(getProperties);
+
+    function getProperties() {
+            Word.run(function (context) {
+                var builtInProperties = context.document.properties;
+                context.load(builtInProperties)
+                return context.sync()
+                    .then(function () {
+                        console.log("Author: " + builtInProperties.author);
+                        console.log("Title: " + builtInProperties.title);
+                        console.log("Application Name: " + builtInProperties.applicationName);
```

Comment:
Labeling is not consistent. Need to be sentence case so change Application Name to Application name.